src/reconstruction/CoreModel.jl

```diff
@@ -402,7 +402,7 @@ end
         model,
         [
             ridx for ridx in 1:n_reactions(model) if
-            any(in.(findnz(model.S[:, ridx])[2], Ref(metabolite_idxs)))
+            any(in.(findnz(model.S[:, ridx])[1], Ref(metabolite_idxs)))
```

Comment:
What exactly is this function doing?

Reply:
Checking which reactions are to be removed (because they touch the metabolite).

Reply:
Oh it removes any reaction that has a metabolite being removed, of course. Is that the default assumption (that one would want all reactions affected to be gone)?

Reply:
yeah. after some thinking we assumed this way to be better, because it prevents breaking semantics of single reactions (e.g. if you have "transport of something" and remove 1 of the 2 metabolites, you suddenly get "infinite supply of something"). This way is more "conservative".